text/0010-dependency-mappings.md

```diff
@@ -11,10 +11,26 @@ Consumers of Paketo buildpacks may be building in an environment where certain d
 ## Detailed Explanation
 
 ### Discovery
-If one or more bindings of kind or type `dependency-mapping` are present, buildpacks should respect any URIs found in the binding. The `sha256` of the dependency should be used to lookup mapped URIs.
+If one or more bindings of kind or type `dependency-mapping` are present, buildpacks should respect any URIs found in the binding. The lookup digest should be included as `checksum` in the form of `<algorithm>:<hash>` (or `sha256` in the form of `<hash>`) of the dependency should be used to lookup mapped URIs.
+
+If the digest is provided in the form of `<hash>`, it will be assumed to be of algorithm type `sha256`.
+
+The dependency mapping implementation in the project will continue to support the use of a `sha256` as long as the `sha256` `buildpack.toml` field is supported; however, users of this feature should ideally use a `checksum`.
 
 ### CNB Bindings Specification Example
```

Comment:
Again, splitting hairs, but I think we should put this specification second and make a note that it's deprecated and we suggest folks use the Service Binding spec for K8s.